cheat/app/src/main/res/layout/item_chat_message_by_me.xml

```diff
@@ -2,7 +2,7 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:id="@+id/item_chat_message_by_me"
```

Comment:
`item_chat_message_by_me.xml` should have id `item_chat_message_by_me` and not `item_chat_message_by_other`